site/content/docs/user/quick-start.md

```diff
@@ -366,8 +366,14 @@ kubeadmConfigPatches:
 {{< /codeFromInline >}}
 
 #### IPv6 clusters
-You can run ipv6 only clusters using `kind`, but first you need to enable ipv6 in your docker daemon by editing `/etc/docker/daemon.json` [as described here][docker enable ipv6].
-Ensure you `systemctl restart docker` to pick up the changes.
+You can run IPv6 only clusters using `kind`, if the host that runs the docker containers support IPv6. Nowadays, all Operating Systems use to have IPv6 enabled by deafult, but you can check it in linux with the following command:
+
+```sh
+sudo sysctl net.ipv6.conf.all.disable_ipv6
+net.ipv6.conf.all.disable_ipv6 = 0
+```
+
+If you are using docker in Windows or Mac, you don't have to do anything, because the containers run inside a VM that already has IPv6 enabled.
```

Comment:
IPv6 clusters don't work on mac. You can't port forward.

Reply:
Confusingly IPv6 Networks do seem fine? but the port forward will fail.

Reply:
AFAIK only linux can run IPv6 clusters even after these changes.

Reply:
I'm confused, the cluster works but with limitations because of the host to container port forwarding?